Gemfile

```diff
@@ -147,10 +147,10 @@ group :development, :test do
   gem "knapsack_pro", "~> 2.16.0" # Help parallelize Ruby spec builds
   gem "pry-byebug", "~> 3.8" # Combine 'pry' with 'byebug'. Adds 'step', 'next', 'finish', 'continue' and 'break' commands to control execution
   gem "rspec-rails", "~> 5.0" # rspec-rails is a testing framework for Rails 3+
-  gem "rubocop", "~> 1.14", require: false # Automatic Ruby code style checking tool
```

Comment:
From what I can see `Super-Linter` doesn't let us specify command-line options for the linters so we can't just run `rubocop --require rubocop-rspec` or similar.